packages/react/src/components/FeatureFlags/index.js

```diff
@@ -29,17 +29,37 @@ const FeatureFlagContext = createContext(GlobalFeatureFlags);
  * along with the current `FeatureFlagContext` to provide consumers to check if
  * a feature flag is enabled or disabled in a given React tree
  */
-function FeatureFlags({ children, flags = {} }) {
+function FeatureFlags({
+  children,
+  flags = {},
+  enableUseControlledStateWithValue,
+  enableV12TileDefaultIcons,
+  enableV12TileRadioIcons,
+  enableV12Overflowmenu,
+  enableTreeviewControllable,
+  enableExperimentalFocusWrapWithoutSentinels,
```

Comment:
Do these need the same defaults as what's defined in the root flag definition?

https://github.com/carbon-design-system/carbon/blob/b9abee7893060affe080b215d9feba8618f9cc10/packages/feature-flags/feature-flags.yml#L25-L52